### PR TITLE
Story 2 game screen

### PR DESCRIPTION
**Pull request by f-gate**
Created splash screen with animated title, play button, background image, city image, & instructions box.

All to be reviewed :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+
```
Comment (hoptond) [inline index.html]: whitespace

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <link rel="stylesheet" href="css/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Play:400,700" rel="stylesheet">
+</head>
+<body>
+    <div id="splash-screen">
+        <img src="img/background.jpg" id="splash-background">
+        <div id="splash-title-wrapper">
+            <h1 id="splash-title">
+                ROCKET MAN
+            </h1>
```
Comment (hoptond) [inline index.html]: should be on one line

File: js/start-game.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    $rocketImg.animate({
+        left: "+=5000",
+    }, 3000)
+
+    setTimeout(function(){
+     $title.fadeIn(50)
+    }
+    , 800)
+
+
```
Comment (hoptond) [inline js/start-game.js]: whitespace

File: js/start-game.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
```
Comment (hoptond) [inline js/start-game.js]: whitespace

File: css/styles.css [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+#splash-open-game {
+    display: block;
+    width: 200px;
+    padding: 10px;
+    margin: 50px auto 50px auto;
+    font-size: 3.2rem;
+    text-align: center;
+    border-style: solid;
+    text-decoration: none;
+    color: white;
```
Comment (hoptond) [inline css/styles.css]: All these should be hex values or RGB. Consistency is important

**Review by hoptond — CHANGES_REQUESTED** [review]
img/fire-png-gif--489.gif shouldn't have two dashes in it

**Review by hoptond — DISMISSED** [review]

File: index.html [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+<head>
+    <meta charset="utf-8">
+    <title>Rocket Man</title>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Play:400,700" rel="stylesheet">
+</head>
+<body>
+    <div id="splash-screen">
+        <img src="img/background.jpg" id="splash-background">
```
Comment (mporam) [inline index.html]: why is this not a background image?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+        <div id="splash-title-wrapper">
+            <h1 id="splash-title">ROCKET MAN</h1>
+            <div id="splash-title-rocket">
+                <img src="img/fire-png-gif-489.gif" id="splash-flames" >
+                <img src="img/rocket-large.png" id="splash-rocket-img">
+            </div>
+        </div>
+        <a href="#" id="splash-open-game">Play Me</a>
```
Comment (mporam) [inline index.html]: use a div here, not an a as it doesnt link anywhere

File: js/start-game.js [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+    $rocketImg.animate({
+        left: "+=5000",
+    }, 3000)
+    setTimeout(function(){
+     $title.fadeIn(50)
+    }
+    , 800)
+})
+
+document.getElementById('splash-open-game').addEventListener("click", function(){
```
Comment (mporam) [inline js/start-game.js]: this means when you click play, the game disappears. thats not very helpful, this should be in the next story

File: js/start-game.js [hunk 9]
```diff
@@ -0,0 +1,15 @@
+$( window ).on( "load", function () {
+    var $rocketImg = $('#splash-title-rocket')
+    var $title = $('#splash-title')
+    $rocketImg.animate({
+        left: "+=5000",
+    }, 3000)
+    setTimeout(function(){
+     $title.fadeIn(50)
+    }
+    , 800)
```
Comment (mporam) [inline js/start-game.js]: comma should be on the line above

File: js/start-game.js [hunk 10]
```diff
@@ -0,0 +1,15 @@
+$( window ).on( "load", function () {
+    var $rocketImg = $('#splash-title-rocket')
```
Comment (mporam) [inline js/start-game.js]: why create variables here? just wasting memory, you can just use the selectors straight up, as you only use each once

File: js/start-game.js [hunk 11]
```diff
@@ -0,0 +1,15 @@
+$( window ).on( "load", function () {
```
Comment (mporam) [inline js/start-game.js]: https://google.github.io/styleguide/jsguide.html
says no space between `function` and `()`

File: js/start-game.js [hunk 12]
```diff
@@ -0,0 +1,15 @@
...
+    var $rocketImg = $('#splash-title-rocket')
+    var $title = $('#splash-title')
+    $rocketImg.animate({
+        left: "+=5000",
+    }, 3000)
+    setTimeout(function(){
```
Comment (mporam) [inline js/start-game.js]: https://google.github.io/styleguide/jsguide.html
says space between `function()` and `{`

**Review by mporam — CHANGES_REQUESTED** [review]

**Review by mporam — APPROVED** [review]

**Pull request merged**